lib/defines.mk

```diff
@@ -38,6 +38,10 @@ ifeq ($(SYSTEM_LIBBPF),y)
 DEFINES += -DLIBBPF_DYNAMIC
 endif
 
+ifeq ($(LIBBPF_V1_COMPLIANT),y)
+DEFINES += -DUSE_LIBBPF_V1_SECTION_NAMES
+endif
```

Comment:
I think we can just set this unconditionally; this will only affect the BPF programs built as part of xdp-tools, and the assumption here is that those will only be used with the same version of the library (which will then understand the new section names straight away)...